### ISSUE TITLE
NVDA reads unparsed MathML leaves if aria-hidden="false" is applied to containing span in IE

### ISSUE THREAD

**Issue by sinabahram** [nvaccess/nvda#7635]
### Steps to reproduce:

1. Open IE on Windows 10
2. Go to this URL: https://pac.bz/downloads/MathMLWithAriaHiddenFalse.html
3. Notice on that page that identical code examples exist, but one of them has aria-hidden="false", which should have no effect, around the span containing the MathML.
4. Read the page with NVDA, and notice that under the "Does Not Work" heading, NVDA reads the leaf nodes of the MathML instead of just reading the parsed MathML correctly as it does under the "Works Great" heading.

### Expected behavior:
Having aria-hidden="false" should have absolutely zero effect on any reading of anything compared to no aria-hidden attribute at all.

### Actual behavior:
NVDA reads the leaf nodes of the MathML.

### System configuration:
NVDA version: next-14463,7dca66c4 Installed
Note: Also confirmed on NVDA 2017.3

### Other questions:

Does the issue still occur after restarting your PC?
yes

Have you tried any other versions of NVDA?
Yes, and see above RE 2017.3

**Comment by NSoiffer** [comment]
# Minor clarification
Actual behavior is that NVDA reads the leaf nodes of the MathML and then properly reads the MathML as math (i.e., you hear the MathML twice, once as kind of garbage as it is just reading the leaf text and then properly). Without the aria-hidden="false", it properly reads just the MathML.

Under configuration, you also have to MathPlayer installed to hear the math read properly.

**Comment by Adriani90** [comment]
This behavior is occuring only when using arrow keys to navigate through the website in browse mode. When I am on the mathematical expression and press nvda+alt+m, NVDA reads the math expression as expected in both cases.
cc: @leonardder

**Comment by Adriani90** [comment]
Support for IE has completely ended in 2022 and the browser is automatically removed from machines that still have it after an Edge update in February 2023. I am closing this as abandoned. Please comment in case you can reproduce this in other browsers as well and we can reopen this issue.
